src/App.js

```diff
@@ -1,15 +1,16 @@
 import React, { Component } from "react";
 
-import "./assets/App.css";
+import "./assets/css/App.css";
+import "./assets/css/App.scss";
 
 class App extends Component {
-    render() {
-        return (
-          <div>
-              <h1>Authors Haven Application!</h1>
-            </div>
-        );
-    }
+  render() {
```

Comment:
Thank you for the good work @BagzieGracious , However, this component should be written as a pure function as our eslintrc specifies.You can write it as 
```
const App = () => (
  <div>
    <h1>Authors Haven Application!</h1>

  </div>
);

```

Reply:
Thank you @CryceTruly for the review, but I have solved the issue.